analyzers/tests/SonarAnalyzer.UnitTest/PackagingTests/CsRuleTypeMapping.cs

```diff
@@ -3690,7 +3690,7 @@ internal static class CsRuleTypeMapping
             //["3690"],
```

Comment:
This same change in VbRuleTypeMapping is missing.

Reply:
Nice catch! Thanks!